x-pack/legacy/plugins/ingest_manager/index.ts

```diff
@@ -19,6 +20,7 @@ import {
 export function ingestManager(kibana: any) {
   return new kibana.Plugin({
     id: 'ingestManager',
+    publicDir: resolve(__dirname, '../../../plugins/ingest_manager/public'),
```

Comment:
What's the reason for needing this line?

Reply:
we need to serve up a local image and it's not yet supported on NP.  you can see the image if you go to the list packages page.